src/main/kotlin/three/consulting/epoc/dto/TimesheetDTO.kt

```diff
@@ -0,0 +1,27 @@
+package three.consulting.epoc.dto
+
+import three.consulting.epoc.entity.Timesheet
+import java.time.LocalDateTime
+import javax.validation.constraints.NotBlank
+
+data class TimesheetDTO(
+    val id: Long? = null,
+    @field:NotBlank val name: String,
+    val description: String? = null,
+    val allocation: Double,
```

Comment:
What is your take on the format for allocation

Reply:
I'd prefer 0-100. I don't know if @robertsiipola has a prefence what he'd like in the frontend side. Anyhoo, if we use 0-100 this should be int and we can validate the field with
```kotlin
@field:Min(0) @field:Max(100) val allocation: Int
```

Reply:
And if you add the Min/Max validation please add a couple tests to see the request fails when the value is outside those boundaries.

Reply:
0-100 is fine with me 👍🏼